workloads/devstack-xenial.yaml

```diff
@@ -57,14 +51,43 @@ runcmd:
     "DATABASE_PASSWORD=\$ADMIN_PASSWORD"
     "RABBIT_PASSWORD=\$ADMIN_PASSWORD"
     "SERVICE_PASSWORD=\$ADMIN_PASSWORD"
-    > {{template "DEVSTACKPATH" .}}/local.conf' {{.User}}
+    > /home/{{.User}}/devstack/local.conf' {{.User}}
   - {{endTaskCheck .}}
-{{ end }}
 
   - {{beginTask . "Stacking the system" }}
-  - su -c "cd {{template "DEVSTACKPATH" .}} && nohup ./stack.sh > /home/{{.User}}/stack.log &" {{.User}}
+  - su -c "cd /home/{{.User}}/devstack && nohup ./stack.sh > /home/{{.User}}/stack.log &" {{.User}}
   - {{endTaskCheck . }}
 
-# Will enable this once the other PR is merged. Very crude this way, but github does not support PR dependencies.
-# {{/*messageHost . "Follow stack.log in $HOME to see output of stacking process" */}}
+  - {{message . "" }}
+  - {{message . "====================================================================" }}
+  - {{message . "" }}
+  - {{message . "devstack initialised with a default local.conf" }}
+  - {{message . "" }}
+  - {{message . "    [[local|localrc]]" }}
+{{with .HostIP}}
+  - {{message $ (printf "    HOST_IP=%s" . ) }}
+{{else}}
+  - {{message . "    HOST_IP=127.0.0.1" }}
+{{end}}
+  - {{message . "    ADMIN_PASSWORD=secret" }}
+  - {{message . "    DATABASE_PASSWORD=\\$ADMIN_PASSWORD" }}
+  - {{message . "    RABBIT_PASSWORD=\\$ADMIN_PASSWORD" }}
+  - {{message . "    SERVICE_PASSWORD=\\$ADMIN_PASSWORD" }}
+  - {{message . "" }}
+  - {{message . "To start devstack with these defaults type" }}
+  - {{message . "" }}
+  - {{message . (printf "    ccloudvm run %s 'cd devstack && ./stack.sh'" .Hostname) }}
```

Comment:
This command is already running on line no. 58. Maybe this should say "if the defaults are of our preference, no action is required and you can follow stack.log in $HOME"